### PR TITLE
Update events.yml

### PR DESCRIPTION
**Pull request by divijjoshi**
Added event.

**Event title:**

**Organizer's name:**

cc @hasgeek/events-review

### CODE REVIEW

File: _data/events.yml [hunk 1]
```diff
@@ -629,3 +629,12 @@
   end_time: 2018-03-11 17:00 
   url: https://fifthelephant.in/2018-mar-dl-bootcamp/
   blurb: Learn theory and practical concepts of building a deep learning solution in the space of computer vision and natural language processing.
+  
+  name: vidhi-bengaluru-book-launch
```
Comment (akshaybhalotia) [inline _data/events.yml]: Add a hyphen (-) and space before `name`. For example, look at line number 624.

Comment (divijjoshi) [reply _data/events.yml]: Thanks Akshay. :) New to this.

File: _data/events.yml [hunk 2]
```diff
@@ -629,3 +629,12 @@
   end_time: 2018-03-11 17:00 
   url: https://fifthelephant.in/2018-mar-dl-bootcamp/
   blurb: Learn theory and practical concepts of building a deep learning solution in the space of computer vision and natural language processing.
+  
+- name: vidhi-bengaluru-book-launch
+  title: "Vidhi Bengaluru Book Launch on Legal Reforms" 
+  city: Bangalore
+  venue: Mother Tekla Auditorium, 143, General KS Thimayya Rd, Bengaluru
```
Comment (akshaybhalotia) [inline _data/events.yml]: Reduce characters to 40, maybe just the auditorium name and city

File: _data/events.yml [hunk 3]
```diff
@@ -629,3 +629,12 @@
...
+  start_time: 2018-02-16 18:00
+  end_time: 2018-03-11 19:30
```
Comment (akshaybhalotia) [inline _data/events.yml]: Check end time, it's in March

File: _data/events.yml [hunk 4]
```diff
@@ -629,3 +629,12 @@
...
   url: https://fifthelephant.in/2018-mar-dl-bootcamp/
   blurb: Learn theory and practical concepts of building a deep learning solution in the space of computer vision and natural language processing.
+  
+- name: vidhi-bengaluru-book-launch
+  title: "Vidhi Bengaluru Book Launch on Legal Reforms" 
+  city: Bangalore
+  venue: Mother Tekla Auditorium, 143, General KS Thimayya Rd, Bengaluru
+  start_time: 2018-02-16 18:00
+  end_time: 2018-03-11 19:30 
+  url: https://www.eventbrite.com/e/vidhi-bangalore-briefing-book-launch-and-panel-discussion-tickets-42976922131?aff=es2
```
Comment (akshaybhalotia) [inline _data/events.yml]: Consider using a shortlink, such as https://bit.ly or https://goo.gl

**Comment by akshaybhalotia** [comment]
Done, thanks @divijjoshi !

**Pull request merged**